os/include/tinyara/arch.h

```diff
@@ -2141,6 +2141,45 @@ int up_rtc_getdatetime(FAR struct tm *tp);
 int up_rtc_settime(FAR const struct timespec *tp);
 #endif
 
+/****************************************************************************
+ * Name: up_register_wakehandler
+ *
+ * Description:
+ *   This function registers a handler to be called when the core wakes up
```

Comment:
Suggest to change as below:
"to be called"-> "for systick compensation"

Reply:
Thank you for your suggestion.
I changed~